src/postgres/src/test/regress/yb_pg_groupingsets_schedule

```diff
@@ -0,0 +1,15 @@
+# src/test/regress/yb_pg_groupingsets_schedule
+#
+####################################################################################################
+# This includes tests for Group
+####################################################################################################
+# Dependencies for `yb_pg_groupingsets`
+test: yb_pg_int8
+
+# Chain of dependencies for `yb_pg_window`
+test: yb_pg_create_function_1
+test: yb_pg_create_type
+test: yb_pg_create_table
+test: yb_pg_copy
+
+test: yb_pg_groupingsets
```

Comment:
Seems like overkill to create a schedule for effectively a single test: can you move this to `yb_pg_misc_serial_schedule`?

Reply:
I've added` yb_pg_limit`,` yb_pg_random`, `yb_pg_groupingsets` to `yb_pg_misc_serial_schedule` in a separate pull request. In that, I've also removed the tests for `yb_pg_aggregates `as mentioned in https://github.com/yugabyte/yugabyte-db/pull/5190#discussion_r462831602

Reply:
See: https://github.com/yugabyte/yugabyte-db/pull/5264